packs/equipment/ghosthands-comet.json

```
@@ -49,18 +47,59 @@
         "quantity": 1,
         "range": 300,
         "reload": {
-            "value": "0"
+            "value": ""
         },
         "rules": [
             {
                 "diceNumber": 1,
                 "dieSize": "d6",
                 "key": "DamageDice",
-                "label": "Bane",
+                "label": "PF2E.WeaponPropertyRune.bane.Name",
                 "predicate": [
                     "target:trait:beast"
                 ],
                 "selector": "damage"
```

```suggestion
                "selector": "{item|id}-damage"
```
Might as well get this selector too while you're at it